framework/widgets/ActiveForm.php

```diff
@@ -30,12 +30,12 @@ class ActiveForm extends Widget
      * @event ActiveFieldEvent an event raised right before rendering an ActiveField.
      * @since 2.1.0
      */
-    const EVENT_BEFORE_FIELD_RENDER = 'beforeFieldRender';
+    const EVENT_BEFORE_FIELD_RENDER = 'widget.form.field.render.before';
```

Comment:
Should it be reflected that it's activeform?

Reply:
I would prefer usage of 'form' and 'field' instead of 'activeform' and 'activefield'.
`ActiveForm` is just the name of the widget we use, which reflects it is a "smart" form. But for the event having 'active' inside its name looks redundant. E.g. "not smart" form is unlikely to trigger event - otherwise it becomes "smart", isn't it?

Reply:
Makes sense.

Reply:
done